errors.go

```diff
@@ -0,0 +1,92 @@
+package webrtc
+
+import (
+	"errors"
+	"fmt"
+)
+
+// Types of InvalidStateErrors
+var (
+	ErrConnectionClosed = errors.New("connection closed")
```

Comment:
I don't feel strongly about this, but what do you think of renaming everything from `Err -> Error`? If you disagree push back, don't feel strongly.

I don't think everyone will ever be confused by Err, but just a little more clear/less clever!

Reply:
It seems golint does have a prefference:
> errors.go:10:2: error var ConnectionClosedError should have name of the form ErrFoo
> errors.go:24:2: error var NoConfigError should have name of the form ErrFoo
> errors.go:38:2: error var CertificateExpiredError should have name of the form ErrFoo
> errors.go:39:2: error var NoTurnCredError should have name of the form ErrFoo
> errors.go:40:2: error var TurnCredError should have name of the form ErrFoo
> errors.go:41:2: error var ExistingTrackError should have name of the form ErrFoo
> errors.go:67:2: error var ModPeerIdentityError should have name of the form ErrFoo
> errors.go:68:2: error var ModCertificatesError should have name of the form ErrFoo
> errors.go:69:2: error var ModRtcpMuxPolicyError should have name of the form ErrFoo
> errors.go:70:2: error var ModIceCandidatePoolSizeError should have name of the form ErrFoo

Yikes!

Reply:
Yeah! I've seen Err in places as well, I'm okay with that and I am sure sean is as well